apps/careers-app/webapp/src/view/vacancies/vacancyDetail.tsx

```
@@ -0,0 +1,1123 @@
+                        color: theme.palette.brand.orange,
+                      }
+                    },
+                    "& a": {
+                      color: theme.palette.brand.orange,
+                      textDecoration: "none",
+                      "&:hover": {
+                        textDecoration: "underline",
+                      }
+                    }
+                  }}
+                  dangerouslySetInnerHTML={{ __html: vacancy.mainContent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)